unstructured/ingest/evaluate.py

```diff
@@ -107,8 +104,7 @@ def measure_text_edit_distance(
         connector = doc.split("/")[0]
         if fn_txt in source_list:  # type: ignore
             output_cct = elements_to_text(elements_from_json(os.path.join(output_dir, doc)))
-            with open(f"{os.path.join(source_dir, fn_txt)}") as f:
-                source_cct = f.read()
+            source_cct = _read_text(os.path.join(source_dir, fn_txt))
```

Comment:
I know mine is based on your outdated version now, but [I added the errors param](https://github.com/Unstructured-IO/evaluations/blob/main/all-doc-types-eval/aggregate-metrics.py#L94) to make sure the entire process doesn't break if an unexpected character is detected in any file. May be helpful for you too